app/src/main/java/io/github/antishake/browser/FileHelper.java

```diff
@@ -49,7 +50,12 @@ public static boolean isDirectory(String path) {
 
   public static List<VideoFileItem> retrieveVideoFiles(String path) {
     File root = new File(path);
-    File[] files = root.listFiles();
+    File[] files = root.listFiles(new FilenameFilter() {
```

Comment:
I think you want to put this filter in retrieveTextFiles() function instead.
Plus, what about filter for video files?